src/main/webapp/index.html

```diff
@@ -131,7 +129,7 @@
 
 <div class="container">
 
-	<h1>CRUD Mission - Thorntail</h1>
+	<h1>Thorntail CRUD Example</h1>
```

Comment:
May be `Thorntail REST CRUD Example` ? Similarly to `Thorntail REST API Level 0 Example`